modules/basics/src/main/java/com/opengamma/strata/basics/market/ObservableKey.java

```diff
@@ -60,6 +60,10 @@
    * @param marketDataFeed  the market data feed that is the source of the observable market data
    * @return the identifier corresponding to this key
    */
-  ObservableId toObservableId(MarketDataFeed marketDataFeed);
+  public abstract ObservableId toObservableId(MarketDataFeed marketDataFeed);
 
+  @Override
+  public default MarketDataId<Double> toMarketDataId(MarketDataFeed marketDataFeed) {
```

Comment:
Maybe this method could return `ObservableId`, avoiding the need for the other method.